### PR TITLE
deposit: url and additional_url in 8564 MARC field

### PR DESCRIPTION
**Pull request by pedrogaudencio**

### CODE REVIEW

File: inspire/base/recordext/fields/inspire.cfg [hunk 1]
```diff
@@ -519,6 +519,8 @@ title_translation:
 
 
 url:
+    schema:
+        {'url': {'type': 'list', 'force': True}}
```
Comment (pedrogaudencio) [inline inspire/base/recordext/fields/inspire.cfg]: @jalavik, amended ;)

File: inspire/modules/deposit/workflows/literature.py [hunk 2]
```diff
@@ -458,7 +458,11 @@ def restructure_ffts(fft):
         # URLs
         # ====
         if 'additional_url' in metadata and metadata['additional_url']:
-            metadata['url'] = {"url": metadata["additional_url"]}
+            if metadata['url']:
```
Comment (jalavik) [inline inspire/modules/deposit/workflows/literature.py]: This line caused: ```KeyError('url')```

File: inspire/base/recordext/fields/inspire.cfg (hunk 1 above)
Comment (jalavik) [inline inspire/base/recordext/fields/inspire.cfg]: @pedrogaudencio As witnessed in #196, be careful when changing the type of the field here as existing submissions probably have the URL value as string.

Comment (jmartinm) [inline inspire/base/recordext/fields/inspire.cfg]: @pedrogaudencio,  Did you take into account @jalavik comment?

Comment (pedrogaudencio) [inline inspire/base/recordext/fields/inspire.cfg]: Yeah, it's covered [here](https://github.com/inspirehep/inspire-next/pull/193/files#diff-3b59804856efabb4ecf3b800ac348ee9R464). ;)

**Comment by jmartinm** [comment]
I think we have been using the ```Signed-off``` too lightly. Please read http://invenio.readthedocs.org/en/latest/developers/git-workflow.html#r2b-commit-message-qa-review-directives

> Signed-off-by
The author says, in essence: “I have carefully implemented the feature without any leftover to-be-fixed places, I have run all code kwalitee checks and all relevant unit and functional tests, and everything is good. To the best of my knowledge, this commit is good to go into the fast merge track”.

This PR has not been even slightly tested, as, in the current state:

1) It produces an error just by submitting the form
2) Completely removes all URLs, not producing any 856 tags

So instead of signed-off there should be a big red sign, saying not tested, please review, test and obviously, don't deploy.

You can see the fixes to make it work here: https://github.com/jmartinm/inspire-next/commit/23c207383962bbf866b7ec2ff693bdbd3bbd360c

**Comment by jmartinm** [comment]
Merged 

6d8b8bb9cf6bbabebcb09ed1c833664661c9def0

**Pull request closed without merging**